shared/desktop/renderer/inputmonitor.js

```diff
@@ -16,38 +16,43 @@ class InputMonitor {
 
   startActiveTimer = () => {
     // wait 5 minutes before adding listeners
+    if (this.activeTimeoutID) window.clearTimeout(this.activeTimeoutID)
     this.activeTimeoutID = window.setTimeout(this.goListening, 300000)
   }
 
   resetInactiveTimer = () => {
```

Comment:
:sheep: you could get rid of this fcn now, but it makes the code easier to follow/understand imo